Lib9c/Action/Interface/IHackAndSlash.cs

```diff
@@ -0,0 +1,9 @@
+using Libplanet.Action;
+
+namespace Nekoyume.Action.Interface
+{
+    public interface IHackAndSlash: IAction
+    {
+
+    }
```

Comment:
This is not important, only for styling.
```suggestion
    public interface IHackAndSlash : IAction
    {
    }
```